content/api-references/events.md

```diff
@@ -143,6 +144,7 @@ The table below shows the documents required to appeal the various CIP rejection
 | `TAX_IDENTIFICATION`    | -                                     | {{<hint danger>}}Required {{</hint>}} | -                                   |
 | `ADDRESS_VERIFICATION`  | {{<hint danger>}}Required {{</hint>}} | -                                     | {{<hint info>}}Optional {{</hint>}} |
 | `DATE_OF_BIRTH`         | {{<hint danger>}}Required {{</hint>}} | -                                     | -                                   |
+| `WATCHLIST_HIT`         | {{<hint danger>}}Required {{</hint>}} | -                                     | -                                   |
```

Comment:
```suggestion
```

Reply:
A watchlist hit requires further review from our KYC team at Alpaca. It typically does not require additional documentation so this line is not needed.